modules/main.js

```diff
@@ -0,0 +1,17 @@
+import BookListApp from './index.js';
+import navLinks from './switch.js';
+import getDate from './datetime.js';
+// instantiate class
+const books = new BookListApp();
+
+books.AppendBook();
+
+books.removeButton();
+
+books.displayBooks();
+
+getDate();
+
+navLinks();
+
+document.addEventListener('DOMContentLoaded', books.displayBooks());
```

Comment:
- [ ] Kindly fix the following behaviour:
   - On page reload when there're books in local storage, each book is rendered twice (See attached below)
   
![image](https://user-images.githubusercontent.com/62290545/193617358-19a028ad-2834-4cf1-8cce-67027fff3f69.png)